runtime/src/bank/tests.rs

```diff
@@ -12665,31 +12665,6 @@ fn test_rehash_accounts_unmodified() {
     assert_eq!(post_bank_hash, prev_bank_hash);
 }
 
-/// Ensure rehash() *does* change the bank hash if accounts are modified
-#[test]
-#[should_panic(expected = "rehashing is not allowed to change the account state")]
```

Comment:
Note to reviewer: without accounts delta hash, this test no longer panicking. The test seems to guard against ledger-tool usage, to ensure `rehash` panic is accounts were modified. Probably need to find different approach to check that?